packages/app-check/src/internal-api.test.ts

```diff
@@ -90,6 +92,7 @@ describe('internal api', () => {
     clearState();
     removegreCAPTCHAScriptsOnPage();
     return deleteApp(app);
+    sinon.restore();
```

Comment:
There should actually be a universal afterEach restore() due to importing `..test/setup` but this doesn't hurt anything.